### DISCUSSION TITLE
How can I hide related content on posts?

### DISCUSSION THREAD

**Discussion by jenkuntz** [CaiJimmy/hugo-theme-stack#758] [Q&A] [answered]
I'd like to hide the Related Content section on posts, but can't seem to find the right parameter or flag to set in the config file. 

In partials > article > components > related-content.html, I can see an "if hidden <> true" type of statement so I assume there *is* some syntax I can put in config to hide this, without copying this code to my layouts and physically commenting it out.

I can't seem to find a similar discussion or question, nor can I see this in the documentation or previous issues. Apologies I it's there and I have missed it.

Thx

**Comment by CaiJimmy** [comment, marked as answer]
I'm afraid that currently there's no parameter to disable it. The easy way is to overwrite that part of the theme, by placing an empty file in `layouts/partials/article/components/related-content.html`

**Reply by jenkuntz** [reply]
Thank you, I'll play around with that option.
